app/components/views/TransactionsPage/index.js

```diff
@@ -1,41 +1,28 @@
 // @flow
-import React from "react";
-import PropTypes from "prop-types";
-import { autobind } from "core-decorators";
-import TabbedPage from "../../TabbedPage";
-import { FormattedMessage as T } from "react-intl";
-import { withTabSlide } from "../../PageTransitions";
-import transactions from "../../../connectors/transactionsPage";
+import Header from "./Header";
+import { RouteTransition } from "shared";
 
-const propTypes = {
-  router: PropTypes.object.isRequired,
-  isTestNet: PropTypes.bool.isRequired,
-};
+const enterLeft = { atEnter: { offset: -100 }, atActive: { offset: 0 }, atLeave: { offset: 100 }, mapStyles: (styles) => ({ transform: `translateX(${styles.offset}%)` })};
+const enterRight = { atEnter: { offset: 100 }, atActive: { offset: 0 }, atLeave: { offset: -100 }, mapStyles: (styles) => ({ transform: `translateX(${styles.offset}%)` })};
+const opts = { stiffness: 150, damping: 15 };
```

Comment:
Same `damping: 20`

Reply:
I can set defaults inside RouteTransition if you want.

Reply:
sure